config/playground/docker/dev/scripts/custom_osd_entry.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+
+import os
+import sys
+
+def run():
+    source_file = "/opt/opensearch-dashboards/extra_config/osd.yml"
+    target_file = "/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml"
```

Comment:
```suggestion
    SOURCE_FILE = "/opt/opensearch-dashboards/extra_config/osd.yml"
    TARGET_FILE = "/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml"
```

Reply:
Yeah, changed to use uppercase for constants in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/d3fb29f28eb993f65f38537b077716d8db2dd0f2)